server/data/data-ldap/src/test/java/org/apache/james/user/ldap/DockerLdapSingleton.java

```diff
@@ -34,7 +34,4 @@
         .password(ADMIN_PASSWORD)
         .build();
 
-    static {
-        ldapContainer.start();
```

Comment:
Instead of removing this, I would more likely go to `LdapGenericContainer` class and change the start method to:

```
public void start() {
        if (!container.isRunning()) {
            container.start();
        }
    }
```
All methods starting the container point to this one. If you do what is above if you have some tests where you don't start it manually, you gonna spend quite some time debugging with the CI to spot and fix them. 

And also with this you are more sure you have only one container running. If you call by mistaking manually the start twice, then you have the same problem as initially

Reply:
Your way is better (y) 
Honestly, I had a worry about "you don't know if that static call is used in some other tests".

Reply:
No worries :)